containers/groovy-examples/docker-compose.yml

```diff
@@ -1,49 +1,17 @@
 version: "3.4"
 
 services:
-  server:
-    image: ghcr.io/deephaven/server:${VERSION:-latest}
-    expose:
-      - '8080'
+  deephaven:
+    image: ghcr.io/deephaven/server-netty:latest
```

Comment:
We want to update to the *non* netty versions.

Reply:
Whoops. That's a typo I repeated through every file. Fixing it now.